src/bios_handler.cpp

```diff
@@ -283,7 +285,9 @@ void IbmBiosHandler::saveAmmToVpd(const std::string& i_memoryMirrorMode)
         (i_memoryMirrorMode == "Enabled" ? constants::AMM_ENABLED_IN_VPD
                                          : constants::AMM_DISABLED_IN_VPD)};
 
-    // TODO: Call write keyword API to update the value in VPD.
+    m_manager->updateKeyword(
+        SYSTEM_VPD_FILE_PATH,
+        vpd::types::IpzData("UTIL", constants::kwdAMM, l_valToUpdateInVpd));
```

Comment:
vpd is not required as this file is under vpd namespace.

Reply:
Ok, I will address